api/v1/runtimecomponent_types.go

```diff
@@ -153,6 +153,10 @@ type RuntimeComponentSpec struct {
 	// Disable information about services being injected into the application pod's environment variables. Default to false.
 	// +operator-sdk:csv:customresourcedefinitions:order=27,type=spec,displayName="Disable Service Links",xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
 	DisableServiceLinks *bool `json:"disableServiceLinks,omitempty"`
+
+	// Tolerations to be added to application pods.
```

Comment:
Suggestion: Tolerations to be added to the application pods. Tolerations allow the scheduler to schedule pods on nodes with matching taints.